images/variables.tf

```diff
@@ -14,6 +14,26 @@ variable "users_account_id" {
 # These parameters have reasonable defaults.
 # ------------------------------------------------------------------------------
 
+variable "administerkmskeys_role_description" {
+  description = "The description to associate with the IAM role (as well as the corresponding policy) that allows sufficient permissions to administer KMS keys in the Images account."
```

Comment:
Again, I'm wondering if we want a role that can administer _all_ KMS keys, or just the one used to encrypt AMIs.

If the former, then the description and default value here should state "...to administer all KMS keys in the ..." for clarity.  Similarly for the description for the `administerkmskeys_role_name` variable.

If the latter then the descriptions also need to change to something like, "...to administer the KMS key used to encrypt AMIs in the Images account."

These changes should roll up into the README as well.

Reply:
As discussed, we are going with administration of all KMS keys in the account.  Clarified in https://github.com/cisagov/cool-accounts/pull/17/commits/97ad10252e39d2b4ea8398a827e5da0aa5453e3e.